canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -167,7 +167,7 @@ class DropDown extends React.Component {
 	handleOnInputChange(evt) {
 		if (evt !== null) {
 			const value = evt;
-			this.props.controller.updatePropertyValue(this.props.propertyId, value);
+			this.props.controller.updatePropertyValue(this.props.propertyId, value, true, "initial_load");
```

Comment:
This function is called on initial_load and also on manual change from carbon's combobox component. We can't differentiate between both actions here. I think its better to not pass any type here.

Reply:
@nmgokhale So it's OK to miss some of` initial_load ` types in this case ?

Reply:
After initially loading `oneofselect_paramDef`, I see `oneofselect_custom_value` is called twice. First time it correctly passes "initial_load" and second time it prints from `handleOnInputChange()`. At this time, its shouldn't pass`initial_load` again. 
![Screen Shot 2022-08-17 at 11 34 43 AM](https://user-images.githubusercontent.com/25124000/185217011-6ed6ea5a-f6d1-449a-8cbb-c1d454bff730.png)